### DISCUSSION TITLE
Missing partial pay/stripe/_checkout_button

### DISCUSSION THREAD

**Discussion by markaharper** [pay-rails/pay#1112] [Q&A] [answered]
Working on installing and setting up Pay Gem with Rails 7.2.1 Ruby 3.3.0.

Getting hit with a 'Missing partial pay/stripe/_checkout_button' error.

Anyone else having this issue?

```
<%= render partial: "pay/stripe/checkout_button", locals: { session: @checkout_session, title: "Checkout"} %>
```

**Comment by excid3** [comment, marked as answer]
This partial was removed. Please follow the Stripe Checkout docs: https://github.com/pay-rails/pay/blob/main/docs/stripe/8_stripe_checkout.md
